src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -587,6 +593,7 @@ def _flexible_server_params(command_group):
                 c.argument('backup_byok_identity', arg_type=backup_identity_arg_type)
                 c.argument('backup_byok_key', arg_type=backup_key_arg_type)
                 c.argument('disable_data_encryption', arg_type=disable_data_encryption_arg_type)
+                c.argument('public_access', arg_type=public_access_arg_type)
```

Comment:
The existing `public_access_arg_type` may not be a suitable one because it's help message says it accepts ip address/`all`/`none`, but in this case, you only allow `Enabled`/`Disabled`
https://github.com/Azure/azure-cli/blob/8b675e42ba3722114cc93df2e313b27ee2dfe044/src/azure-cli/azure/cli/command_modules/rdbms/_params.py#L366-L373

Reply:
Add another arg type for update server command